test/functional/issues_controller_test.rb

```diff
@@ -149,4 +149,126 @@ def test_show_without_closed_subtasks
       assert_select "tr#issue-#{closed_issue.id}", count: 0
     end
   end
+
+  def test_render_issue_tree_dir_file_model
```

Comment:
It had 3 tests that failed, because of routing , I just added " only: [] " to my code : resources :issue_view_columns, only: [] do
    member do
      patch 'update_collapsed_ids'
    end
  end